en/docs/install-and-setup/upgrading-wso2-is-as-key-manager/upgrading-from-is-km-570-to-5100.md

```diff
@@ -12,43 +12,42 @@ The following information describes how to upgrade your **WSO2 API Manager (WSO2
 
     -   If you wish to upgrade your APIM environment from **API-M 2.6.0 to 3.1.0** , which is using the **internal** WSO2 Identity Server (**WSO2 IS**) **capabilities** , follow the instructions in [Upgrading API-M from 2.6.0 to 3.1.0](../upgrading-wso2-api-manager/upgrading-from-260-to-310.md).
 
-Upgrade WSO2 API-M **from WSO2 API-M 2.6.0 to 3.1.0** when using **WSO2 IS** as the **Key Manager:** as mentioned in below.
+When **WSO2 Identity Server (WSO2 IS)** is the **Key Manager**,
 
-Follow the instructions mentioned in [Upgrading API-M from 2.6.0 to 3.1.0](../upgrading-wso2-api-manager/upgrading-from-260-to-310.md), 
-but **instead of step 2 - (5)**, which explains how to migrate the WSO2 API-M Identity components, 
-follow the WSO2 Identity Server 5.10.0 documentation to migrate the WSO2 Identity Server (WSO2 IS) from version 5.7.0 to 5.10.0.
-[Migrate the WSO2 Identity Server (WSO2 IS) from version 5.7.0 to 5.10.0](https://is.docs.wso2.com/en/5.10.0/setup/migrating-to-5100/)
+Follow the instructions in [Upgrading API-M from 2.6.0 to 3.1.0](../upgrading-wso2-api-manager/upgrading-from-260-to-310.md),
+**instead of step 2 - (5)**, which explains how to migrate the WSO2 API-M Identity components, 
```

Comment:
Where do we guide to follow the next steps of step 2 -(5) of "Upgrading API-M from 2.6.0 to 3.1.0" doc ? I guess it should be either at the end of this page.

Reply:
This is page is something like alteration of step 2 -(5) of "Upgrading API-M from 2.6.0 to 3.1.0" doc. So that order should be mentioned clearly

Reply:
You mean like 
1. follow the  "Upgrading API-M from 2.6.0 to 3.1.0" doc up to step 2  - (4)
2. follow the IS migration doc 
3. follow the  "Upgrading API-M from 2.6.0 to 3.1.0" doc starting step 2 - (6)
Is that?

Reply:
Done

Reply:
And I assume you are planning to send another PR to do the discussed changes (pointing to IS KM doc page) in "Upgrading API-M from 2.6.0 to 3.1.0" doc page.